stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -413,7 +406,7 @@ public static void asyncUnimplementedUnaryCall(MethodDescriptor<?, ?> methodDesc
     responseObserver.onError(Status.UNIMPLEMENTED
         .withDescription(String.format("Method %s is unimplemented",
             methodDescriptor.getFullMethodName()))
-        .asException());
+        .asRuntimeException());
```

Comment:
What's the difference here? Should we always onError SRE?

Reply:
I prefer SRE